static/js/i18n/i18n.test.ts

```diff
@@ -128,30 +128,30 @@ test("formatNumber", async () => {
       value: 0.25,
       unit: "%",
       expected: {
-        de: "0,25\xa0%",
+        de: "0,25 %",
```

Comment:
@beets Do you know why this was \xa0 before? This seems related to the change of icu4c-data change in package-lock.json?